### DISCUSSION TITLE
Add Codespaces from Student Developer Pack to a free Organization

### DISCUSSION THREAD

**Discussion by IMGROOT2** [community/community#38122] [Codespaces]
Hello, community!

Earlier this month, I was fortunate to receive the GitHub Student Developer Pack, which gives free "Pro" level access to Codespaces. I want to enable this feature on my free organization, which I am the owner of. I do not know how to activate my Codespaces access in my organization's Billing section of settings. There's nothing in the docs, either.

This might be new to the Developer Pack, but I wanted to know if anyone here knows what to do. Codespaces will benefit my organization greatly and allow my peers and me to collaborate on our project even better.

Thank you for the help!

**Comment by ghost** [comment]
+1 sorting by newest, this feature must have just come out, but I'm experiencing the same thing, there is simply no way to enable access to Codespaces via the Student Developer pack 
![image](https://user-images.githubusercontent.com/112320303/199685596-28468e69-84ad-4f49-b154-81be3a2d2ee2.png)

The link that reads "Get direct access on the website GitHub Codespaces" just links to this informational page: https://github.com/features/codespaces

**Reply by ghost** [reply]
I also opened a ticket directly with GitHub support here: https://support.github.com/contact/education. It autoclosed my ticket with an irrelevant canned response, but I was able to reopen it.

**Comment by grimsteel** [comment]
It seems like this was a mistake - it's not showing up in the pack anymore...
![image](https://user-images.githubusercontent.com/115331356/199833687-72b184f4-7839-4223-b3b4-2db189d93fe1.png)
This would have been a really nice feature for the Student Developer pack (many people want it - #18209)
Hopefully Github _is_ considering adding it in the future...

**Comment by ossanna16** [comment]
Hi there @IMGROOT2 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by grimsteel** [comment]
It looks like we now have access to this! (It probably relates to #38697)
[![image](https://user-images.githubusercontent.com/115331356/200951962-8be2d6db-0604-4ae2-82c7-6b016f307d70.png)](https://github.com/features/codespaces#is-codespaces-available-for-students)
